oakerr/language.go

```diff
@@ -8,34 +8,34 @@ import (
 type Language int
 
 var (
-	currentLanguage Language = EN
+	// CurrentLanguage is the current language for error and log strings
+	CurrentLanguage Language
 )
 
-func SetLanguage(l Language) {
-	currentLanguage = l
-}
-
 // SetLanguageString parses a string as a language
 func SetLanguageString(language string) error {
 	language = strings.ToUpper(language)
 	switch language {
 	case "EN", "ENGLISH":
-		currentLanguage = EN
+		CurrentLanguage = ENG
 	case "DE", "GERMAN", "DEUTSCH":
-		currentLanguage = DE
+		CurrentLanguage = DEU
 	case "JP", "JAPANESE", "日本語":
-		currentLanguage = JP
+		CurrentLanguage = JPN
 	default:
 		return InvalidInput{InputName: language}
```

Comment:
I know this is unchanged from previous versions but perhaps it makes sense to enhance this error type for uses where we have an easily enumerable set of accepted inputs.

In other words how do we feel about adding a new field to invalid input that is an slice of valid things and print it if applicable.